### PR TITLE
Use tasks instead of events

### PR DESCRIPTION
**Pull request by paulrouget**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/23900)

### CODE REVIEW

**Comment by paulrouget** [comment]
Hmm, forgot to remove the Event type.

**Comment by jdm** [comment]
@bors-servo retry
* #23290

File: support/hololens/ServoApp/BrowserPage.cpp [hunk 1]
```diff
@@ -313,9 +286,7 @@ void BrowserPage::OnSurfaceManipulationDelta(
   auto y = e.Position().Y;
   auto dx = e.Delta().Translation.X;
   auto dy = e.Delta().Translation.Y;
-  Event event = {{Event::SCROLL}};
-  event.scrollCoords = {dx, dy, x, y};
-  SendEventToServo(event);
+  RunOnGLThread([=] { mServo->Scroll(x, y, dx, dy); });
```
Comment (Manishearth) [inline support/hololens/ServoApp/BrowserPage.cpp]: This should be `dx, dy, x, y`

**Pull request merged**